daisy_workflows/image_build/debian/fai_config/packages/GCE_SPECIFIC

```diff
@@ -0,0 +1,3 @@
+# To avoid entropy issues when intances boot
+PACKAGES install
+haveged
```

Comment:
This should be in the upstream GCE config right?

Reply:
I am trying to convince the debian-cloud team to install haveged in GCE images right now. Let's see the outcome.

Reply:
Cool thanks- is there a thread on that?

Reply:
No, we were discussing that on IRC. Bastian told me to file a critical bug against `src:linux` but he doesn't want to install `haveged` for now. He also mentioned that Azure already provides a significant chunk of random data, via DMI. So can we keep this current solution until we get some progress in this front?

Reply:
Yup no problemo- just wondering whats going on there.